README.md

```diff
@@ -50,7 +50,17 @@ Run express backend server, make sure there is a build of the client to serve (`
 npm run serv
 ```
 
-> :warning: TODO add unit testing.
+Same as above, but with live reload.
+
+```bash
+npm run devserv
+```
+
+Run tests.
+
+```bash
+npm run coverage
+```
 
 > :warning: TODO add hmr to the express app, looking at vite-node, node-hmr etc..
```

Comment:
Do we still want this ? is Hot Module Reload useful enough to try find a solution? I'd spent a bit of time trying to find a way to get it working, but no dice, so am wondering if we still want it.